src/pages/Address/Cells.tsx

```diff
@@ -188,21 +200,37 @@ const Cell: FC<{ cell: LiveCell }> = ({ cell }) => {
     index: `0x${cell.cellIndex.toString(16)}`,
   }
 
+  const handleClick = () => {
+    history.push(link)
+  }
+
   return (
-    <li key={cell.txHash + cell.cellIndex} className={styles.card}>
+    <li
+      key={cell.txHash + cell.cellIndex}
+      className={classNames(styles.card, styles.pointer)}
+      onClick={handleClick}
+      onKeyDown={handleClick}
+    >
       <h5>
-        <a href={link}>{title}</a>
-
+        {title}
         <button type="button" className={styles.copy} data-detail={JSON.stringify(outPoint)} onClick={handleCopy}>
           <CopyIcon />
         </button>
-        <span title={`${ckb} CKB`}>{`${ckb} CKB`}</span>
+        <span className={styles.pointer} title={`${ckb} CKB`}>{`${ckb} CKB`}</span>
       </h5>
       <div className={styles.content}>
         {typeof icon === 'string' ? <img src={icon} alt={assetName ?? 'sudt'} width="40" height="40" /> : null}
         {icon && typeof icon !== 'string' ? icon : null}
         <div className={styles.fields}>
-          <div className={styles.assetName}>{assetName}</div>
+          {assetLink ? (
+            <div className={styles.assetName} onClick={handleAssetNameClick} onKeyDown={undefined}>
+              <Link to={assetLink} title={assetName}>
```

Comment:
`handleAssetNameClick` can be set on `Link` if `e.preventDefault` is removed at https://github.com/Magickbase/ckb-explorer-frontend/pull/303/files#diff-70402f47afde8be9bd94344462430a7909cd50bb5001d7da78b771a95bd752a6R142

`prevent default` disables the default behavior of the element, e.g. jump on clicking a link